src/roi/__tests__/borderLengths.test.ts

```diff
@@ -35,6 +37,6 @@ test('border lengths property 4x4', () => {
   ]);
   const roiMapManager = fromMask(mask);
   const rois = roiMapManager.getRois();
-  expect(rois[0].borderLengths).toStrictEqual([2, 1]);
-  expect(rois[1].borderLengths).toStrictEqual([2, 2]);
+  expect(rois[0].borderIDs[0].id).toStrictEqual(-2);
+  expect(rois[0].borderIDs[0].length).toStrictEqual(1);
```

Comment:
You are no longer testing the same thing

Reply:
It still tests functionality of the borderLengths though. The way borderIDs look now is completely different so I had to change the tests.